sdk/src/types/block/semantic.rs

```diff
@@ -7,6 +7,7 @@ use core::fmt;
 use hashbrown::{HashMap, HashSet};
 use primitive_types::U256;
 
+use super::payload::signed_transaction::TransactionHash;
```

Comment:
Merge

Reply:
I really wish vscode would stop doing that

Reply:
Mine doesn't 🤷🏻‍♂️